### PR TITLE
New Rule: Attachment: Adobe image lure with suspicious link from first time sender

### PR DESCRIPTION
**Pull request by morriscode**
284 messages found ,2 Already Flagged. All bad. No FP's in testing.

### CODE REVIEW

**Comment by cameron-dunn-sublime** [comment]
/update-test-rules

Test message

File: detection-rules/attachment_adobe_image_lure_fts.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Adobe image lure with with suspicious link from first time sender"
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure_fts.yml:1]: do we want to add a check for a suspicious link? right now we're just looking for phrases and existence of a link

File: detection-rules/attachment_adobe_image_lure_fts.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+description: "Detects Adobe phishing messages with an Adobe logo attached, with suspicious link language
+  from a first-time sender."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp'))) == 0
+  and length(body.links) >0
+  and all(body.links, .display_text is null)
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure_fts.yml:10]: this is surprising. would have thought the phrases below would be on the display text

File: detection-rules/attachment_adobe_image_lure_fts.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  and any(attachments,
+       beta.logo_detect(.).brand.confidence in ("high")
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure_fts.yml:12]:
```suggestion
      beta.logo_detect(.).brand.confidence in ("high")
```

File: detection-rules/attachment_adobe_image_lure_fts.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  and (
+          (
+              sender.email.domain.root_domain in $free_email_providers
+              and sender.email.email not in $sender_emails
+          )
+          or (
+              sender.email.domain.root_domain not in $free_email_providers
+             and sender.email.domain.domain not in $sender_domains
+          )
+  )
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure_fts.yml:28]: should we exclude emails from Adobe?

File: detection-rules/attachment_adobe_image_lure_fts.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Adobe image lure with with suspicious link from first time sender"
+description: "Detects Adobe phishing messages with an Adobe logo attached, with suspicious link language
+  from a first-time sender."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp'))) == 0
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure_fts.yml:8]: we could totally add another version of this rule looking for branded PDFs now too 😎

File: detection-rules/attachment_adobe_image_lure_fts.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/attachment_adobe_image_lure_fts.yml:1]: we don't check other rules with similar intent.. eg DocuSign. Testing validates it's uneeded.

File: detection-rules/attachment_adobe_image_lure_fts.yml (hunk 2 above)
Comment (morriscode) [reply detection-rules/attachment_adobe_image_lure_fts.yml:10]: The image is hyperlinked so no need for display text

File: detection-rules/attachment_adobe_image_lure_fts.yml (hunk 4 above)
Comment (morriscode) [reply detection-rules/attachment_adobe_image_lure_fts.yml:28]: Doesn't seem to be required. Adobe isn't really notorious for unsolicited spam, and legit emails won't match this criteria.

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**